### ISSUE TITLE
[Bug] <Write short description here>

### ISSUE THREAD

**Issue by slughappy1** [tachiyomiorg/tachiyomi#4998]
**PLEASE READ THIS**

I acknowledge that:

- I have updated:
  - To the latest version of the app (stable is v0.10.12)
  - All extensions
- I have tried the troubleshooting guide: https://tachiyomi.org/help/guides/troubleshooting-problems/
- If this is an issue with an extension, that I should be opening an issue in https://github.com/tachiyomiorg/tachiyomi-extensions
- I have searched the existing issues and this is new ticket **NOT** a duplicate or related to another open issue
- I will fill out the title and the information in this template

Note that the issue will be automatically closed if you do not fill out the title or requested information.

**DELETE THIS SECTION IF YOU HAVE READ AND ACKNOWLEDGED IT**

---

## Device information
* Tachiyomi version: 0.10.12
* Android version: 10
* Device: Galaxy Tab A (SM-T510)

## Steps to reproduce
1. More, Settings, Security
2. Turn on Require unlock
3. Lock when idle doesn't matter as it happens on all values
4. Get out of the app by either going home or locking the device
5. Open app
6. App closes without fully opening

### Expected behavior
Require entering unlock pin same as lock screen

### Actual behavior
The app doesn't open and instead immediately closes. 

## Other details
This feature works just fine on my phone with a finger print reader, but not on my tablet which doesn't have a finger print reader. Once this feature is enabled, the only way to get the application to be able to open again is to clear app data. I am unsure how to dump crash logs as the application cannot open until I clear the app data.

If you're experiencing crashes, share the crash logs from More → Settings → Advanced → Dump crash logs.
